### PR TITLE
feat: README template

### LINKED ISSUE #9: readme template/structure
the readme for each example should have a uniform structure, 
same headings e.g. 
```md

# Title 

Description

Link to blogpost

## Overview

Maybe architecture diagram, more detailed explanation than in description, links to relevant resources.

### Prerequisites

what you need to run this

## Develop

instructions how to run example locally

## Deploy

instructions or “next steps” to run example in 
cloud

## Run tests

## Contribute

```

### PR DESCRIPTION
**Pull request by vinckr**
A template for all READMEs in the examples repo. 

This should be the general structure for all READMEs in the repo, so it is easy to navigate the different examples and also easy to add new ones (possibly in an more automated way down the line). 

closes https://github.com/ory/examples/issues/9

### CODE REVIEW

**Review by gen1us2k — DISMISSED** [review]
LGTM

File: _common/README [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!-- README Template for ory/examples. -->
+<!-- Use the following template for all READMEs in ory/examples. -->
+
+# Title 
+
+<!--Add one sentence describing the project, usecase and key services used. -->
+<!--Add a link to post on the Ory Blog as well as other platforms (dev.to, ...). -->
```
Comment (kevgo) [inline _common/README]:
```suggestion
<!-- Add one sentence describing the project, usecase, and key services used. -->
<!-- Add a link to post on the Ory Blog as well as other platforms (dev.to, ...). -->
```

File: _common/README [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Overview
+
+<!-- Optional: Add an diagram if using multiple services or a more detailed explanation than in description or links to other relevant resources. -->
```
Comment (kevgo) [inline _common/README]:
```suggestion
<!-- Optional: Add a diagram if using multiple services or a more detailed explanation than in description or links to other relevant resources. -->
```

File: _common/README [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+### Prerequisites
+
+<!--Add a list of software needed to run the example, ideally with links to the installation page or installation (e.g. brew) commands. -->
```
Comment (kevgo) [inline _common/README]:
```suggestion
<!-- Add a list of software needed to run the example, ideally with links to the installation page or installation (e.g. brew) commands. -->
```

File: _common/README [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Develop
+
+<!-- Add a numbered list of commands needed to run the example locally or with Ory Cloud playground. -->
+
+### Environmental Variables
+
+<!-- Optional: If needed add instructions on how to use environmental variables. -->
```
Comment (kevgo) [inline _common/README]:
```suggestion
<!-- Describe needed environment variables + instructions on how to set them up. -->
```

File: _common/README [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+# Title 
+
+<!--Add one sentence describing the project, usecase and key services used. -->
+<!--Add a link to post on the Ory Blog as well as other platforms (dev.to, ...). -->
+
+## Overview
+
+<!-- Optional: Add an diagram if using multiple services or a more detailed explanation than in description or links to other relevant resources. -->
+
+### Prerequisites
```
Comment (kevgo) [inline _common/README]: I could see the prerequisites making more sense as a subsection under `## Develop` than under `## Overview`.

File: _common/README [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+<!-- Add a numbered list of commands needed to run the example locally or with Ory Cloud playground. -->
+
+### Environmental Variables
+
+<!-- Optional: If needed add instructions on how to use environmental variables. -->
+
+### Run tests
+
+<!-- Add a numbered list of commands needed to run tests locally. -->
```
Comment (kevgo) [inline _common/README]: Why a numbered list? There are often several tests that one could run, e.g. unit tests, end-to-end tests, documentation tests, all of the above tests, etc. Those are typically not run in any particular order.

File: _common/README [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+## Deploy
+
+<!-- Optional: Add a list of steps needed to use the example with Ory services selfhosted.-->
+<!-- Optional: Add a list of steps needed to use the example with Ory Cloud developer project.-->
+
+## Contribute
+
+Feel free to [open a discussion](https://github.com/ory/examples/discussions/new) to provide feedback or talk about ideas, or [open an issue](https://github.com/ory/examples/issues/new) if you want to add your example to the repository or encounter a bug.
```
Comment (kevgo) [inline _common/README:37]: I have seen standards to wrap text at 80 characters elsewhere at Ory. Does this standard also apply to this repo?

**Review by kevgo — COMMENTED** [review]
Thanks for doing this, I like it! Some fine-tuning in the comments below.

File: _common/README (hunk 7 above)
Comment (vinckr) [reply _common/README:37]: you are right, we still need to apply the standard templates to this repo and add the prettier styles
https://github.com/ory/meta/blob/master/package.json
https://github.com/ory/meta

I think we can commit this though and then later on format everything.

**Pull request merged**